src/lib/integration_tests/tests/platform_orchestrator/reset_ml_feed_cache_test.rs

```diff
@@ -0,0 +1,385 @@
+    },
+};
+
+const INDIVIDUAL_TEMPLATE_WASM_PATH: &str =
+    "../../../target/wasm32-unknown-unknown/release/individual_user_template.wasm.gz";
+const POST_CACHE_WASM_PATH: &str =
+    "../../../target/wasm32-unknown-unknown/release/post_cache.wasm.gz";
+
+const USER_INDEX_WASM_PATH: &str =
+    "../../../target/wasm32-unknown-unknown/release/user_index.wasm.gz";
+const PF_ORCH_WASM_PATH: &str =
+    "../../../target/wasm32-unknown-unknown/release/platform_orchestrator.wasm.gz";
+
+fn individual_template_canister_wasm() -> Vec<u8> {
+    std::fs::read(INDIVIDUAL_TEMPLATE_WASM_PATH).unwrap()
+}
+
+fn user_index_canister_wasm() -> Vec<u8> {
+    std::fs::read(USER_INDEX_WASM_PATH).unwrap()
+}
+
+fn post_cache_canister_wasm() -> Vec<u8> {
+    std::fs::read(POST_CACHE_WASM_PATH).unwrap()
+}
+fn pf_orch_canister_wasm() -> Vec<u8> {
+    std::fs::read(PF_ORCH_WASM_PATH).unwrap()
+}
+
+// TODO: remove this when removing the update_last_access_time API from PF Orch
+
+#[test]
+fn reset_ml_feed_cache_test() {
+    let (pocket_ic, known_principal) = get_new_pocket_ic_env();
+    let platform_canister_id = known_principal
+        .get(&KnownPrincipalType::CanisterIdPlatformOrchestrator)
+        .cloned()
+        .unwrap();
+
+    let super_admin = known_principal
+        .get(&KnownPrincipalType::UserIdGlobalSuperAdmin)
+        .cloned()
+        .unwrap();
+
+    let application_subnets = pocket_ic.topology().get_app_subnets();
+
+    let charlie_global_admin = get_mock_user_charlie_principal_id();
+    let alice_principal_id = get_mock_user_alice_principal_id();
+    let bob_principal_id = get_mock_user_bob_principal_id();
+    let dan_principal_id = get_mock_user_dan_principal_id();
+
+    pocket_ic
+        .update_call(
+            platform_canister_id,
+            super_admin,
+            "add_principal_as_global_admin",
+            candid::encode_one(charlie_global_admin).unwrap(),
+        )
+        .unwrap();
+
+    let user_index_canister_id: Principal = pocket_ic
+        .update_call(
+            platform_canister_id,
+            super_admin,
+            "provision_subnet_orchestrator_canister",
+            candid::encode_one(application_subnets[1]).unwrap(),
+        )
+        .map(|res| {
+            let canister_id_result: Result<Principal, String> = match res {
+                WasmResult::Reply(payload) => candid::decode_one(&payload).unwrap(),
+                _ => panic!("Canister call failed"),
+            };
+            canister_id_result.unwrap()
+        })
+        .unwrap();
+
+    for i in 0..50 {
+        pocket_ic.tick();
+    }
+
+    // upgrade pf_orch
+
+    let platform_orchestrator_init_args = PlatformOrchestratorInitArgs {
+        version: "v1.0.0".into(),
+    };
+    pocket_ic
+        .upgrade_canister(
+            platform_canister_id,
+            pf_orch_canister_wasm(),
+            candid::encode_one(platform_orchestrator_init_args).unwrap(),
+            Some(super_admin),
+        )
```

Comment:
Why are we upgrading the platform orchestrator canister in the tests?

Reply:
no specific reason. simply copied from a previous test.

Reply:
will remove